src/models/Settings.js

```diff
@@ -104,41 +103,39 @@ define([
       },
 
       // OAUTH2
+      'clientId': 'string',
```

Comment:
we can discuss online, but why are we changing this file?

Reply:
I just move around properties definitions to sections they suppose to be. feels a bit easier to read.